presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java

```diff
@@ -168,6 +175,61 @@ public void testNormalQuery()
         assertEquals(queryCompletedEvent.getStatistics().getTotalRows(), expectedCompletedPositions);
     }
 
+    @Test
+    public void testPrepareAndExecute()
+            throws Exception
+    {
+        final String selectQuery = "SELECT count(*) FROM lineitem WHERE shipmode = ?";
+        final String prepareQuery = "PREPARE stmt1 FROM " + selectQuery;
```

Comment:
Nit: maybe just name this `stmt` (remove the `1`) since we only have one prepared statement in this test